packages/labs/ssr-client/src/lib/hydrate-lit-html.ts

```diff
@@ -163,7 +163,12 @@ export const hydrate = (
       }
       // Create a new ChildPart and push it onto the stack
       currentChildPart = openChildPart(rootValue, marker, stack, options);
-      rootPart ??= currentChildPart;
+      // Using nullish logical assignment below can cause some minifier to move
```

Comment:
```suggestion
      // Using nullish logical assignment below can cause next.js's swc to move
```

More specific, so we know what to check when we're thinking about undoing this later